.github/workflows/tests.yml

```diff
@@ -39,6 +41,9 @@ jobs:
       - name: Composer validate
         run: composer validate --strict --no-check-lock
 
+      - name: Flex Plugin
+        run: composer global config --no-plugins allow-plugins.symfony/flex true
```

Comment:
do we still need this when composer.json already allows the plugin?

Reply:
Yeah, I tried it with just the composer.json file but since it's a global dependency you need to add this as well, otherwise it'll fail